snapcraft/tests/test_yaml.py

```diff
@@ -550,12 +557,23 @@ def test_config_snap_environment(self, mock_snapdir):
         self.assertTrue(len(paths) > 0,
                         'Expected LD_LIBRARY_PATH to be in environment')
 
-        arch = snapcraft.common.get_arch_triplet()
         for expected in ['foo/lib', 'foo/usr/lib', 'foo/lib/{}'.format(arch),
                          'foo/usr/lib/{}'.format(arch)]:
-            self.assertTrue(expected in paths,
-                            'Expected LD_LIBRARY_PATH to include "{}"'.format(
-                                expected))
+            self.assertTrue(
+                expected in paths,
+                'Expected LD_LIBRARY_PATH in {!r} to include {!r}'.format(
```

Comment:
Perhaps `Expected LD_LIBRARY_PATH ({!r}) to include {!r}` would make more sense here?